### ISSUE TITLE
Soft Incompatibility with RimHud

### ISSUE THREAD

**Issue by MadDove** [Solid-Wires/rimworldmod-pawnkindracediversification#27]
When using rimhud, pawns that get their race changed are still counted as another race. Specifically, their rimhud identifier says that they are human when they are another alien. This appears just cosmetic, as their stats are alien stats, but it is a bit annoying.

**Comment by MadDove** [comment]
An example of this is an Anty being spawned. The rimhud says they are human, however all their stats are Anty stats.

**Comment by Solid-Wires** [comment]
Yeah, this issue does exist. It's not something I can fix on my end without aggressive transpiler operations.
This chain of comments should be considered:
![image](https://user-images.githubusercontent.com/72309175/140812305-75eb7e63-9fc2-4a0b-8d74-8b4bb477d705.png)

For further context, this is the language translation string (in English) that pawn.Named("PAWN") and its consequent arguments are being pushed into after translation:
![image](https://user-images.githubusercontent.com/72309175/140815285-37216923-70eb-484f-9605-4b0caca6c69d.png)

**Comment by Solid-Wires** [comment]
Neronix explains this issue better and more concisely in RimHud's issues board on page 9:
![image](https://user-images.githubusercontent.com/72309175/140813718-7996aacc-158a-4166-a602-9654aa161791.png)
